mod_reforged/hooks/config/strings.nut

```diff
@@ -2225,13 +2234,13 @@ foreach (vanillaDesc in vanillaDescriptions)
 			{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"Immediately upon killing an opponent, if you move into the tile of the killed target, the [Action Point|Concept.ActionPoints] cost of the next skill used is " + ::MSU.Text.colorPositive("halved") + "."
+					"Immediately after killing an opponent, if you move into the tile of the killed target, the [Action Point|Concept.ActionPoints] cost of the next skill used is " + ::MSU.Text.colorPositive("halved") + "."
 				]
 			},
 			{
 				Type = ::UPD.EffectType.Active,
 				Description = [
-					"Unlocks the [Gain Ground|Skill+rf_gain_ground_skill] skill which, immediately after killing an adjacent target, allows you to move into their tile ignoring [Zone of Control|Concept.ZoneOfControl] with reduced [Action Point|Concept.ActionPoints] cost and [Fatigue|Concept.Fatigue] cost of movement."
+					"Unlocks the [Gain Ground|Skill+rf_gain_ground_skill] skill which, immediately after killing an adjacent opponent, allows you to move into their tile ignoring [Zone of Control|Concept.ZoneOfControl] and with reduced [Action Point|Concept.ActionPoints] and [Fatigue|Concept.Fatigue] cost."
```

Comment:
Let's decide if we want target or opponent. I'm fine with both but perhaps `opponent` sounds more mercenary like. In the perk's tooltip you changed `enemy` to `target` in this PR. Whatever you end up deciding to go with make sure that both the passive/active part of this perk tooltip and the actual perk's effect tooltip have the same.

Reply:
Perk descriptions within strings tend to be a bit more flowery and less precise, so that's why I used opponent there and target within the Gain Ground skill itself. I'll change target to opponent within Gain Ground skill if you prefer though.

Reply:
I see then it's fine like this the way you have.